package.json

```diff
@@ -6,7 +6,9 @@
   "scripts": {
     "start": "gulp serve",
     "start:prod": "sudo systemctl start sexxi.xyz",
-    "build": "gulp build"
+    "build": "gulp build",
+    "pretest": "npm install -g gulp",
```

Comment:
What is `pretest` for :stuck_out_tongue: I am pretty sure `npm install` will install `gulp` locally